### DISCUSSION TITLE
I cannot run the "run_evaluation" function

### DISCUSSION THREAD

**Discussion by TLampr** [DevSkillsHQ/discusssions#38] [Help] [answered]
Hi,

I am trying to run the "run_evaluation" function in the **Unsupervised IRT Model Parameter Estimation** but I am missing the PERSON_PARAM_TRUE_TOKEN and the ITEM_PARAM_TRUE_TOKEN environment variables. 

Is there something I am missing while setting up the environment or is this function only supposed to be used after I submit my solution?

I built my environment using Gitpod.

Thank you in advance!

**Comment by fenske** [comment, marked as answer]
Hi @TLampr. I'm Anton from Alva. The run_evaluatoin script is running every time you push to a branch on your remote repository which will trigger a build running the script.

When working on the task, just focus on hitting the described metric values described in the README.

Hope this clarifies it.

BR,
Anton.
